tests/src/tracing/tracevalidation/rundown/rundown.csproj

```diff
@@ -11,8 +11,8 @@
     <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\..\</SolutionDir>
     <CLRTestKind>BuildAndRun</CLRTestKind>
     <DefineConstants>$(DefineConstants);STATIC</DefineConstants>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <CLRTestPriority>0</CLRTestPriority>
+    <GCStressIncompatible>true</GCStressIncompatible>
```

Comment:
I believe that this should be GCStress compatible.  The assembly unload events should be present even with GCStress enabled because the assemblies will be loaded regardless.